src/Sarif.Driver/Sdk/MultithreadedAnalyzeCommandBase.cs

```diff
@@ -242,7 +242,7 @@ public virtual TContext InitializeContextFromOptions(TOptions options, ref TCont
             context.ResultKinds = options.ResultKinds;
             context.FailureLevels = options.FailureLevels;
             context.OutputFileOptions = options.OutputFileOptions.ToFlags();
-            context.MaxFileSizeInKilobytes = options.MaxFileSizeInKilobytes;
+            context.MaxFileSizeInKilobytes = options.MaxFileSizeInKilobytes != null ? options.MaxFileSizeInKilobytes.Value : context.MaxFileSizeInKilobytes;
```

Comment:
>MaxFileSizeInKilobytes [](http://example.com/codeflow?start=94&length=22)

This pattern is required in order to permit a combination of options- and context-specified configuration. We need to make a general scrub for this pattern I've implemented for max file size only.